### ISSUE TITLE
doesn't work 

### ISSUE THREAD

**Issue by chunsx** [imacrosid/shopee-live-support#1]
Failed Get List Voucher.
![image](https://github.com/imacrosid/shopee-live-support/assets/157081337/8ffcfbb4-9b59-4f18-a44e-52ca2ec61c5e)

**Comment by imacrosid** [comment]
Yes because shopee update, i've updated version but i don't want share again because i caught someone trying to get profit from selling/using my work.
